spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/advisor/DocumentRetrievalAdvisor.java

```diff
@@ -21,11 +32,9 @@
 /**
  * Title Document retrieval advisor.<br>
  * Description Document retrieval advisor.<br>
- * Created at 2024-08-27 10:47
  *
  * @author yuanci.ytb
- * @version 1.0.0
- * @since jdk8
+ * @since 2024/8/16 11:29
```

Comment:
It is more common to use version numbers instead of dates, `@since 1.0.0-Snapshot`